post-merge

```diff
@@ -1,3 +1,34 @@
 #!/bin/sh
-composer install --no-dev --no-suggest
+NO_COLOR="\033[0m"
+RED="\033[38;5;009m"
+GREEN="\033[38;5;010m"
+YELLOW="\033[38;5;011m"
+ORANGE="\033[38;5;214m"
+LIGHTPURPLE="\033[38;5;177m"
+PURPLE="\033[38;5;135m"
+CYAN="\033[38;5;014m"
+LIGHTBLUE="\033[38;5;39m"
+BLUE="\033[38;5;75m"
+DARKBLUE="\033[38;5;33m"
+LIGHTGRAY="\033[38;5;252m"
+DARKGRAY="\033[38;5;242m"
+BRIGHTRED="\033[91m"
+BOLD="\033[1m"
+
+echo "\n${GREEN}post merge hook start${NO_COLOR}\n"
+
+# we check if vendor/bin/phpunit is present.
+# If yes this means we are in dev mode.
+PHPUNIT="vendor/bin/php-cs-fixer"
+
+if [ -x "$PHPUNIT" ]; then
+    echo "\n${ORANGE}Dev mode detected${NO_COLOR}\n"
+    composer install --no-suggest
+else
+    echo "\n${ORANGE}--no-dev mode detected${NO_COLOR}\n"
+    composer install --no-dev --prefer-dist --no-suggest
+fi
 php artisan migrate
+
+
+echo "\n${RED}post merge hook finish${NO_COLOR}\n"
```

Comment:
Why red?

If you want to test for non-zero return codes and show red/green that might work.